src/main/java/uk/ac/ebi/embl/template/TemplateEntryProcessor.java

```diff
@@ -260,6 +260,8 @@ private void replaceOrganismToken(TemplateVariables templateVariables, TemplateP
                 }else {
                     String delimitedKey = StringBuilderUtils.encloseToken(TemplateProcessorConstants.ORGANISM_TOKEN);
```

Comment:
- Only comment is that we might want to create a function for this that we would call twice:
```
String delimitedKey = StringBuilderUtils.encloseToken(TemplateProcessorConstants.ORGANISM_TOKEN);
                    template = new StringBuilder(template.toString().replace(delimitedKey, scientificName));
```